docs/modules/general/pages/implementation-details/mfe-content-loading/index.adoc

```diff
@@ -0,0 +1,64 @@
+
+Prior to any routing events, Shell UI initializes router to reflect workspace configuration. Based on the data from backend services about the current workspace, Shell UI creates routing rules to use during the runtime.
+
+Let's take a workspace using user management Application as an example. Each Application in workspace configuration has to have a path defined. This path is used for routing and accessing remote entry file of Application's Microfrontend. Assuming the path is `/user`, Shell UI will create a routing rule specifying that whenever url of the page changes to one starting with `/user` it has to load the Microfrontend's module associated with user management Application.
+
+After routing rules are successfully created, each registered Application should have a Route registered in Shell UI's router. Each Route has a **loadChildren method, which is called whenever route gets activated** and other properties used for route matching.
+
+The **loadChildren method is responsible for loading the remote module into the Shell UI's memory using module-federation**. When remote module is loaded successfully, it is used by the router to display module's content a page content pane.
+
+==== Slot Component mechanism (Remote Component)
+Remote Component loading is handled by Slot Component. Overall mechanism of Remote Component loading was explained in the xref:architecture-overview/remoteComponents.adoc[Remote Components documentation]. In the context of content loading its important to note the following aspects:
+
+* Remote Component **loading process starts when Slot Component is rendered** on a page.
+* Remote Component **exposed content is loaded via module-federation** (similar to module content type). Then once its loaded successfully, it is **rendered onto a page within the Slot Component via dynamic content creation mechanism dependent on the chosen expose method**.
+
+=== Expose method
+Expose method is a factor that is taken into consideration by the respective loading mechanism (Shell UI router or Slot Component) to ensure correct content loading of the content. For module loading, the loadChildren method and router configuration are impacted. For Remote Component loading, different mechanism of dynamic content creation is chosen. For information on specific expose method impacts, please refer to the following documents:
+
+* xref:implementation-details/mfe-content-loading/webcomponents.adoc[Webcomponent method]
+
+=== Module federation loading explained
+Independent on the content type and expose method, OneCX platform will load exposed content using module-federation. During this step exposed content is loaded using the file specified in webpack configuration for respective content. In the code snippet below, `./src/main.ts` file would be used for the module and `./src/app/remotes/my-comp/my-comp.component.main.ts` file for Remote Component loading.
```

Comment:
refer/link to module-federation(?)

Reply:
I'll place a TODO for it since I am not completely sure which article should be referenced.